src/main/java/uk/gov/hmcts/darts/transcriptions/component/TranscriberTranscriptsQuery.java

```diff
@@ -10,4 +10,7 @@ public interface TranscriberTranscriptsQuery {
 
     List<TranscriberViewSummary> getTranscriberTranscriptions(Integer userId);
 
+    List<Integer> getAuthorisedCourthouses(Integer userId, Integer roleId);
+
+    Integer getTranscriptionsCountForCourthouses(List<Integer> courthouses, Integer transcriptionStatusId, int i);
```

Comment:
Should the 3rd param be renamed to be `int userId` in keep with the implementing class?
Maybe the 1sp param should be `courthouseIds`?